web-app/yarn.lock

```diff
@@ -4538,10 +4538,10 @@ devlop@^1.0.0, devlop@^1.1.0:
   dependencies:
     dequal "^2.0.0"
 
-dezalgo@^1.0.4:
-  version "1.0.4"
-  resolved "https://registry.yarnpkg.com/dezalgo/-/dezalgo-1.0.4.tgz#751235260469084c132157dfa857f386d4c33d81"
-  integrity sha512-rXSP0bf+5n0Qonsb+SVVfNfIsimO4HEtmnIpPHY8Q1UCzKlQrDMfdobr8nJOOsRgWCyMRqeSBQzmWUMq7zvVig==
+dezalgo@1.0.3:
+  version "1.0.3"
```

Comment:
Is this intentional downgrade or to make it align with `formidable`?

Reply:
It was not intentional, "formidbale", downgraded it, should be fixed now.